bin/capsule-certs-generate

```diff
@@ -31,6 +31,7 @@ CONFIG_CONTENT = <<EOF
 :answer_file: #{answers_file.path}
 :installer_dir: #{INSTALLER_DIR}
 :installer_name: 'foreman-installer'
+:product_name: 'Katello'
```

Comment:
I see your point; however, I don't see an instance where this is the case looking through the PR. If you did need to refer to Katello during a Capsule install, you'd likely need to refer to the Capsule as well which would mean you'd need to define product_name twice?